src/client/utils/tracking/adTraction.ts

```diff
@@ -0,0 +1,93 @@
+import md5 from 'md5'
+import { TypeOfContract } from 'data/graphql'
+import { OfferData } from 'pages/OfferNew/types'
+import {
+  isBundle,
+  isNorwegian,
+  isYouth,
+  isDanishAccidentBundle,
+  isStudentOffer,
+  isDanishTravelBundle,
+} from 'pages/OfferNew/utils'
+import { NoComboTypes, DkBundleTypes } from './tracking'
+
+type TypeOfContractExcludedUnused = Exclude<
+  TypeOfContract | NoComboTypes | DkBundleTypes,
+  'DK_ACCIDENT' | 'DK_ACCIDENT_STUDENT' | 'DK_TRAVEL' | 'DK_TRAVEL_STUDENT'
+>
```

Comment:
Is this a place where a comment "why" these are unused should be added?

That's a bit unclear at least to me reading this.

Reply:
Good point 🙌